packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -629,6 +659,30 @@ export class TokenDetectionController extends StaticIntervalPollingController<To
     return slicesOfTokensToDetect;
   }
 
+  #getConvertedStaticMainnetTokenList(): TokensChainsCache {
+    let data: TokenListMap = {};
+    for (const [key, value] of Object.entries(STATIC_MAINNET_TOKEN_LIST)) {
+      data = {
+        ...data,
+        [key]: {
+          name: value?.name,
```

Comment:
One question here how come value can be nullable ?

Reply:
Good call 🙏  will update that!